jenkins/shared-library/vars/codeCoverage.groovy

```diff
@@ -50,6 +50,10 @@ void verifyCodeCoverageResult(String tool, Integer minimumCodeCoverage) {
 			logger.logInfo("Minimum code coverage set in pom is ${readJacocoCoverageLimit()}")
 			runScript('mvn jacoco:check@jacoco-check')
 		},
+		'jacoco-gradle': { Integer target ->
+			runScript('gradle jacocoLogTestCoverage')
+			runScript('gradle jacocoTestCoverageVerification')
+		},
```

Comment:
Does it need the target variable?
because I didn't see it being used.

Reply:
yea, the variable is not used.  I guess I can rename it to __ as a placeholder.

Reply:
I can't use "_" in the parameter name, or else the linter fails 😅 
https://codenarc.org/codenarc-rules-naming.html#parametername-rule

Which do you prefer
1. rename var from ``target`` to  ``notUsed``
2. leave it as is
3. update the default linting rules.

Reply:
let's go for option 2.